docs/getting-started.md

```diff
@@ -0,0 +1,119 @@
+# Getting Started
+
+Welcome to the Getting Started guide for the **refarch-templates**. This guide is
+organized into sections covering **Frontend/Web Component**, **Backend/EAI**, and
+**Stack**, along with a general setup and requirements that apply to all templates.
+
+The purpose of these templates is to provide developers with an out-of-the-box
+solution for building full-stack applications.
+
+## General
+
+To use the refarch-templates, you need to have Docker installed on your
+machine. Inside the `stack` folder, you will find a `docker-compose.yml`
+file that will spin up everything needed for local development.
+
+When developing locally, the stack should always be running.
+
+### Download Templates
+
+You can download the templates or clone them directly from the GitHub
+repository.
+
+```bash
+git clone https://github.com/it-at-m/refarch-templates.git
+```
+
+Once the templates are available locally, copy the
+folders you need into your own project directory. For example, if you want
+to work with the backend, you should copy the `refarch-backend` folder.
+
+**Important**: You also need to copy the Stack folder to your project once.
+
+## Stack
+
+The stack includes:
+
+### Keycloak
+
+- **Local Instance**: Set up and run a Keycloak instance locally.
+
+### Keycloak Migration
+
+- **Migration Tool**: Utilize a migration tool that executes scripts upon startup.
+- **Script Location**: All necessary scripts are located in the
+`stack/keycloak` folder.
+Extensions to Keycloak should be implemented through these scripts.
+
+### PostgreSQL
+
+- **Local Instance**: Set up and run a PostgreSQL instance locally.
+
+### PGAdmin
+
+- **Local Instance**: Run a pre-configured instance of PgAdmin that connects to
+your local PostgreSQL database.
+
+### API Gateway
+
```

Comment:
Do we need such specific information? Or is Maven 3.9 or even Maven 3 enough information?

Reply:
but it our standard maven, so we now that it works with it. Does it work with maven 3.8? maven 3.9.1? 3.7? i didnt check and i dont think that there is any reason to check it.